backendAS/.gitignore

```diff
@@ -2,4 +2,5 @@ songs.csv
 *.jpg
 songImages
 commentImages
-comments.csv
\ No newline at end of file
+comments.csv
+*.csv
```

Comment:
NBD: missing a new line at the end of the document.  Some operating systems will have issues with some operations on files without a carriage line at the end, such as unix `cat`

The standard comes from here, and you can find reasons via google: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

Reply:
this is really helpful!!